player.js

```diff
@@ -46,11 +67,12 @@ app.post('/request', (req, res) => {
 
 app.get('/playlist', (req, res) => res.json(playlist))
 
-let server = app.listen(3000, function () {
+const port = argv.p ? argv.p : 3000
+let server = app.listen(port, function () {
   let host = server.address().address;
   let port = server.address().port;
 
-  console.log('Example app listening at http://%s:%s', host, port);
+  console.log('Karakuni listening at http://%s:%s', host, port);
```

Comment:
Karakuri (though I do also like the name "Karakuni", we could do awesome things in the country of karas :))